### PR TITLE
Plugins: Reduxify PluginActivateToggle progress

### PR DESCRIPTION
**Pull request by tyxla**
This PR updates the `PluginActivateToggle` component to use Redux for determining whether the plugin is being (de)activated or not, instead of using the old `PluginsLog` store.

Part of #24180.

#### Changes proposed in this Pull Request

* Plugins: Reduxify PluginActivateToggle progress

#### Testing instructions

* Get a Jetpack site for testing.
* Go to `/plugins/hello-dolly/:site` and install the plugin if it's not installed.
* Activate and deactivate the plugin from the toggle.
* Verify that while (de)activating, the toggle gets disabled.
* Verify all tests still pass.

#### Note

We could be importing the notice actions from the plugins state constants file that defines them, but I'm leaving that for another PR as it affects other locations, too.

### CODE REVIEW

File: client/my-sites/plugins/plugin-activate-toggle/index.jsx [hunk 1]
```diff
@@ -146,8 +143,19 @@ PluginActivateToggle.defaultProps = {
...
 
-export default connect( null, {
-	recordGoogleEvent,
-	recordTracksEvent,
-	togglePluginActivation,
-} )( localize( PluginActivateToggle ) );
+export default connect(
+	( state, { site, plugin } ) => {
+		const pluginStatus = getStatusForPlugin( state, site.ID, plugin.id );
+		const inProgress =
```
Comment (flootr) [inline client/my-sites/plugins/plugin-activate-toggle/index.jsx:149]: Would it make sense to move that to a new selector instead?

**Review by flootr — APPROVED** [review]
works well 👍🏻

File: client/my-sites/plugins/plugin-activate-toggle/index.jsx (hunk 1 above)
Comment (tyxla) [reply client/my-sites/plugins/plugin-activate-toggle/index.jsx:149]: Sure, that's a good call. I'll do that in a follow-up PR after this one and #47921 land.

Comment (tyxla) [reply client/my-sites/plugins/plugin-activate-toggle/index.jsx:149]: Addressed in #47951.

**Pull request merged**